client.go

```diff
@@ -568,6 +568,21 @@ func (c *Client) Create(cfg Function) (err error) {
 		return
 	}
 
+	// Load the created function that was just written to disc
```

Comment:
Why do we need to load function config during `create`? Shouldn't it be alread most recent spec version?

Reply:
No - that is part of the issue. On creation, the version has been set to 0.0.0. I've just pushed some changes that simplify this so that we don't need to read and rewrite the file in create, so this code is gone now.